LayoutTests/fast/mediastream/keep-microphone-muted-on-uninterruption.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html> <!-- webkit-test-runner [ AllowTestOnlyIPC=true ] -->
```

Comment:
This could just be on for all WebKitTestRunner WKWebViews.  This will get annoying really quickly.